README.md

```diff
@@ -103,12 +103,17 @@ logging into Bluemix
 Now that you have your IBM ID and the `cf` command-line tool (see above),
 you can log into Bluemix and the deploy your app.
 
-First you should tell the `cf` command which environment you want to operate
+1. `bx login -sso -a http://api.ng.bluemix.net` (For people with @ibm.com accounts)
```

Comment:
You haven't explained yet what the `bx` command is or how to install it.  And it's only ever referenced in these 4 lines, not clear if maybe the user doesn't need to use `cf` if they have `bx`, or what.  For reference, I have no idea what the `bx` command is myself!

Reply:
For the ibm accounts, you need to generate a token using the sso since I had trouble connecting directly to the bluemix.